kimchi/src/circuits/constraints.rs

```diff
@@ -736,20 +736,25 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
 
         let lookup_domain_size = {
             // First we sum over the lookup table size
+            let mut has_table_with_id_0 = false;
```

Comment:
Taste: I don't like using mutable values, but it avoids iterating a second time in the vector. We could use a `any`.